source/tools/testConsistency/dotnetWPF-renderer/test_app.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) Microsoft Corporation. All rights reserved. 
+# Licensed under the MIT License. 
+import unittest
+import json
+from app import app
+from copy import deepcopy
+
+
+POST_URL = 'http://127.0.0.1:5000/'
+TEST_PARTIAL_RESULT_URL = 'http://127.0.0.1:5000/test'
+
+class TestFlaskApi(unittest.TestCase):
+    def setUp(self):
+        #self.backup_items = deepcopy(app.items)  # no references!
+        self.app = app.test_client()
+        self.app.testing = True
+
+    def testEmptyData(self):
+        jsonFile = open('../../../../samples/v1.0/Scenarios/ActivityUpdate.json')
```

Comment:
>('../../../../samples/v1.0/Scenarios/ActivityUpdate.json') [](start = 23, length = 58)

This would also come in from a test execution parameter or something to allow the test to run from a location other than the current location of the source file here